test/tests/rabbitmq-tls/run.sh

```diff
@@ -13,10 +14,10 @@ RUN set -eux; \
 		-key /certs/ca-private.key \
 		-out /certs/ca.crt \
 		-days $(( 365 * 30 )) \
-		-subj '/CN=lolca'; \
+		-subj "/CN=$cname"; \
```

Comment:
The CA needs a different subject from the cert or `openssl verify` errors out:

```console
+ openssl verify -CAfile /certs/ca.crt /certs/cert.crt
CN = rabbitmq-container-9033-24410
error 18 at 0 depth lookup: self-signed certificate
error /certs/cert.crt: verification failed
```

I tried `$cname-CA` and the `openssl` parts worked, but the RabbitMQ test still fails. 😢 I then added `-ssl_dist_opt server_cacertfile /certs/ca.crt` to `tests/rabbitmq-tls/rabbitmq-env.conf`, but that didn't help.

<details><summary>Log</summary>
<p>

```console
$ ./tests/rabbitmq-tls/run.sh rabbitmq:3.13-rc
...
2023-06-28 21:21:53.499527+00:00 [info] <0.257.0>  cookie hash    : L9yhZwh3R+2yr4j+gxDpKg==
...
2023-06-28 21:21:55.566893+00:00 [info] <0.729.0> started TCP listener on [::]:5672
2023-06-28 21:21:55.568759+00:00 [info] <0.749.0> started TLS (SSL) listener on [::]:5671
 completed with 3 plugins.
2023-06-28 21:21:55.647632+00:00 [info] <0.658.0> Server startup complete; 3 plugins started.
2023-06-28 21:21:55.647632+00:00 [info] <0.658.0>  * rabbitmq_prometheus
2023-06-28 21:21:55.647632+00:00 [info] <0.658.0>  * rabbitmq_web_dispatch
2023-06-28 21:21:55.647632+00:00 [info] <0.658.0>  * rabbitmq_management_agent
++ eval 'rabbitmq-diagnostics check_port_connectivity'
+++ rabbitmq-diagnostics check_port_connectivity
+++ docker run -i --rm --link rabbitmq-container-27405-11238 -e ERLANG_COOKIE librarytest/rabbitmq-tls-server:rabbitmq-3.13-rc rabbitmq-diagnostics --node rabbit@rabbitmq-container-27405-11238 check_port_connectivity
Error: unable to perform an operation on node 'rabbit@rabbitmq-container-27405-11238'. Please see diagnostics information and suggestions below.

Most common reasons for this are:

 * Target node is unreachable (e.g. due to hostname resolution, TCP connection or firewall issues)
 * CLI tool fails to authenticate with the server (e.g. due to CLI tool's Erlang cookie not matching that of the server)
 * Target node is not running

In addition to the diagnostics info below:

 * See the CLI, clustering and networking guides on https://rabbitmq.com/documentation.html to learn more
 * Consult server logs on node rabbit@rabbitmq-container-27405-11238
 * If target node is configured to use long node names, don't forget to use --longnames with CLI tools

DIAGNOSTICS
===========

attempted to contact: ['rabbit@rabbitmq-container-27405-11238']

rabbit@rabbitmq-container-27405-11238:
  * connected to epmd (port 4369) on rabbitmq-container-27405-11238
  * epmd reports node 'rabbit' uses port 25672 for inter-node and CLI tool traffic
  * TCP connection succeeded but Erlang distribution failed
  * suggestion: check if the Erlang cookie is identical for all server nodes and CLI tools
  * suggestion: check if all server nodes and CLI tools use consistent hostnames when addressing each other
  * suggestion: check if inter-node connections may be configured to use TLS. If so, all nodes and CLI tools must do that
   * suggestion: see the CLI, clustering and networking guides on https://rabbitmq.com/documentation.html to learn more


Current node details:
 * node name: 'rabbitmqcli-938-rabbit@52a76a80b8e2'
 * effective user's home directory: /var/lib/rabbitmq
 * Erlang cookie hash: L9yhZwh3R+2yr4j+gxDpKg==
```

</p>
</details> 

Side note is that these can stay as single quotes since the variable is evaluated at the heredoc level first.